web/pandas/about/team.md

```diff
@@ -36,6 +36,14 @@ If you want to support pandas development, you can find information in the [dona
     {% endfor %}
 </div>
 
+## Diversity and Inclusion
+
+> _pandas_ expressly welcomes and encourages contributions from anyone who faces under-representation, discrimination in the technology industry
+> or anyone willing to increase the diversity of our team.
+> We have identified the visible diversity gap and obstacles in the open source community and we are proactive in increasing the diversity of our team.
```

Comment:
maybe `diversity gaps` or `visible gaps and obstacles in sustaining diversity and inclusion in the open-source communities`.
I think adding something along the lines of `sustaining D&I` should be inlcuded to highlight that our efforts are not one-off, but intended for long term.
WDYT?